### ISSUE TITLE
Workout better semantic versioning pattern

### ISSUE THREAD

**Issue by ryanolee** [Bluetel-Solutions/webpack-manifest-bundle#4]
Currently it is hard to follow what versions of the repo support which twig versions

**Comment by alexwilson** [comment]
This wasn't ever really a consideration, IIRC the `Twig_Extension` pattern was supported in both v1 & v2 which were supported by Symfony 2.x.

Newer versions of Symfony (3.3 and beyond?) [support webpack natively](https://symfony.com/blog/new-in-symfony-3-3-manifest-based-asset-versioning) and the purpose of this bundle was to backport that.

For simplicity it's probably best to add some documentation with a migration path to the native functionality, and bumping the current version to 1.0.0 - to encourage further API-surface-creep. (which apparently we never got around to!)

**Comment by alexwilson** [comment]
For what it's worth https://github.com/bluetel/webpack-manifest-bundle/releases/tag/0.7.2 deserves merging back into trunk!

**Comment by ryanolee** [comment]
Sounds like a plan. I will be away for a week but look to clean this up and further action this when I am back :D

**Comment by alexwilson** [comment]
Have a lovely break! 😄
